src/machines/authMachine.ts

```diff
@@ -254,12 +254,12 @@ function getCookieByName(cname: string): string | null {
 async function getAndSyncStoredToken(input: {
   token?: string
 }): Promise<string> {
-  // dev mode
-  const VITE_KITTYCAD_API_TOKEN = env().VITE_KITTYCAD_API_TOKEN
```

Comment:
The "VITE" prefix should not exist in the codebase aside from the interop between environment and application in `src/env.ts`, so I decided to rename this temporary variable to `localToken`.